src/rules/css/detectBadMozBinding.js

```diff
@@ -1,12 +1,12 @@
 import * as messages from 'messages';
```

Comment:
I like that our JS files now aren't `camelCase` or whatever that's called, can we do the same for CSS?